app/services/policies/policies_list_service.rb

```diff
@@ -4,16 +4,19 @@
 require 'set'
 
 class Policies::PoliciesListService
+
+  class PoliciesListServiceError < RuntimeError; end
+
   def self.call(account, builtin: true)
     list = PolicyList.new
-    if builtin
-      response = ::JSONClient.get(ThreeScale.config.sandbox_proxy.apicast_registry_url)
-      return unless response.ok?
-
-      list.merge!(PolicyList.from_hash(response.body['policies']))
+    begin
```

Comment:
```ruby
def self.call(account, builtin: true)
  list = PolicyList.new
  list.merge!(PolicyList.from_hash(fetch_policies_from_apicast)) if builtin
  list.merge!(PolicyList.new(account.policies)) if account.provider_can_use?(:policy_registry)
  list.to_h
rescue PoliciesListServiceError => error
  Rails.logger.error { error }
end
```

Reply:
Ah that's nicer, it has to return `nil` tho. I'll change it.